apps/prairielearn/src/ee/pages/lti13CourseNavigation/lti13CourseNavigation.ts

```diff
@@ -53,6 +54,7 @@ async function courseInstancesAllowedToLink({
   const course_instances = await selectCourseInstancesWithStaffAccess({
     course,
     authzData,
+    // TODO: Why is "Previewer" included here? And why is this "Student Data Viewer" and not "Student Data Editor"?
```

Comment:
👀 I don't understand the current values.

EDIT: I guess these aren't doing what I thought they were. I thought these were the permissions that were checked on the course instances that were returned, but it seems like they're checked on the `authzData` for the current course? This API feels either wrong or misleading to me.

It's not clear to me why `requiredRole` is needed here at all. I would expect a function like `selectCourseInstancesWithStaffAccess` to simply give me back the course instances that the given user has staff access to, and if they don't have any permissions, this would just be an empty list. Maybe I'm thinking about this wrong?